### PR TITLE
refactor: pass mirror env from bin, add --china options

### PR DESCRIPTION
**Pull request by dead-horse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,13 +22,51 @@
...
+Can specify one or more: npm install ./foo.tgz bar@stable /some/folder
+If no argument is supplied, installs dependencies from ./package.json.
+
+Options:
+
+  --production: won't install devDependencies
+  --save, --save-dev, --save-optional: save installed dependencies into package.json
+  -g, --global: install devDependencies to global directory which specified in '$npm config get prefix'
+  -r, --registry: specify custom registry
+  -c, --china: specify in china, will automatically using chinses registry and mirrors
```
Comment (fengmk2) [inline README.md]: China registry mirror and other binary mirrors

File: README.md [hunk 2]
```diff
@@ -22,13 +22,51 @@
...
+```
+
+## Use as Lib
+
+### Install
+
+```
+$npm install npminstall --save
```
Comment (fengmk2) [inline README.md]: $ npm

File: lib/config.js [hunk 3]
```diff
@@ -22,10 +27,8 @@ const config = {
     ELECTRON_MIRROR: 'https://npm.taobao.org/mirrors/electron/',
     SASS_BINARY_SITE: 'https://npm.taobao.org/mirrors/node-sass',
     npm_config_fse_binary_host_mirror: 'https://npm.taobao.org/mirrors/fsevents/',
-    // show node-pre-gyp http info
-    // like "node-pre-gyp http GET https://npm.taobao.org/mirrors/fsevents/v1.0.6/fse-v1.0.6-node-v46-darwin-x64.tar.gz"
-    npm_config_loglevel: 'http',
   },
+  chineseRegistry: 'https://registry.npm.taobao.org/',
```
Comment (fengmk2) [inline lib/config.js]: remove last / char

**Comment by dead-horse** [comment]
fixed

**Comment by fengmk2** [comment]
1.1.0

**Pull request merged**